modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -71,21 +73,22 @@ image::rtf-ingress-endpoint.png[A host and path are set for the application's en
 . To preview the endpoint, click the generated preview link.
 . To add additional endpoints, click *+ Add Endpoint*.
 
-=== Specify the Network Configuration for the Application (for VMs/Bare Metal)
+=== Configure a Public Endpoint for the Application for VMs/Bare Metal
 
-If you are using Runtime Fabric on VMs/Bare Metal, you must xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. 
+Use the following steps to configure a public endpoint and network configuration for an application deployed to Runtime Fabric for VMs/Bare Metal.
 
-[NOTE]
-To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the SAN certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.
+You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure your application is configured to accept connections on port 8081.
```

Comment:
You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure that your application is configured to accept connections on port 8081.